scipy/stats/_stats_mstats_common.py

```diff
@@ -5,208 +5,16 @@
 from ._stats_pythran import siegelslopes as siegelslopes_pythran
 from scipy.stats import _stats_py
```

Comment:
```suggestion
```
If CI is otherwise successful, I'd suggest committing this with `[skip ci]` before merge to resolve the lint error.